LibCarla/source/carla/ros2/Exception.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2017 Computer Vision Center (CVC) at the Universitat Autonoma
+// de Barcelona (UAB).
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "carla/Exception.h"
+#include <asio.hpp>
+
+#ifdef ASIO_NO_EXCEPTIONS
+
+namespace asio {
+namespace detail {
+
+  template <typename Exception>
+  void throw_exception(const Exception& e) {
+    carla::throw_exception(e);
+  }
+
+  template void throw_exception<std::length_error>(const std::length_error &);
+  template void throw_exception<asio::service_already_exists>(const asio::service_already_exists &);
```

Comment:
Add parameter names please

Reply:
Done. also the other exception functions in the other  file